app/pages/blackout/sponsorshop.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
LOL yeah, I named the folder Sponsorshop and instead of renaming the folder, the intellectual in me just keep importing from Sponsorshop